contracts/apps/AppProxy.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity 0.4.15;
+pragma solidity 0.4.18;
```

Comment:
should't be  +pragma solidity ^0.4.18; ?

Reply:
Yes. All files that are used as interfaces have the `^`, files that will be deployed have a compiler version fixed for security reasons. In the case of aragon apps all have to be fixed

Reply:
https://gist.github.com/renexdev/a6325c0d1ec90a7bbd81898e05dbaf50